compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -599,11 +599,13 @@ execBuild projectOpts opts mbOutFile incrementalBuild initPkgDb =
                             (toNormalizedFilePath' $ fromMaybe ifaceDir $ optIfaceDir opts)
                             (FromDalf False)
                     dar <- mbErr "ERROR: Creation of DAR file failed." mbDar
-                    let fp = targetFilePath $ unitIdString (pkgNameVersion pName pVersion)
+                    fp <- targetFilePath relativize $ unitIdString (pkgNameVersion pName pVersion)
                     createDarFile fp dar
             where
-                targetFilePath name = fromMaybe (distDir </> name <.> "dar") mbOutFile
-
+                targetFilePath rel name =
+                  case mbOutFile of
+                    Nothing -> pure $ distDir </> name <.> "dar"
+                    Just out -> rel out
```

Comment:
I don't think we need to pass `relativize` as an argument, do we?
```suggestion
                    Just out -> relativize out
```

Reply:
looks like we do, I think because the `where` clause is not relative to the lambda.